documentation/service-rules/ecf1/participants.md

```diff
@@ -0,0 +1,567 @@
+
+When a participant is registered in the system, they are assigned various attributes that define their status, relationship with schools and providers, and other important information:
+
+### Schedule
+
+The `schedule_identifier` attribute defines the training schedule for the participant. Examples include:
+
+- `ecf-standard-january`: Standard ECF schedule starting in January
+- `ecf-standard-september`: Standard ECF schedule starting in September
+
+The schedule determines the timeline for training activities and assessments.
+
+### Training Status
+
+The `training_status` attribute indicates the current training state of the participant. Possible values include:
+
+- `active`: Participant is actively engaged in training
+- `deferred`: Participant has temporarily paused their training
+- `withdrawn`: Participant has permanently left the training programme
+
+This status is updated when actions like defer, withdraw, or resume are performed on a participant.
+
+### Status
+
+The `participant_status` attribute indicates the administrative status of the participant. Possible values include:
+
+- `active`: Participant is currently active or has completed a training programme
+- `leaving`: Participant is leaving a training programme
+- `left`: Participant has left a training programme
+- `joining`: Participant is joining a training programme
+- `withdrawn`: Participant has withdrawn from a training programme
+
+This status may differ from the training status and also reflects the administrative state rather than the training state.
+
+### URN (Unique Reference Number)
+
+The `school_urn` attribute is the unique identifier for the school where the participant is registered. This is a 6-digit number assigned to educational establishments in England and Wales.
+
+### Delivery Partner ID
+
+The `delivery_partner_id` attribute identifies the delivery partner responsible for providing training to the participant. This is a UUID that links to a delivery partner record.
+
+### TRN (Teacher Reference Number)
+
+The `teacher_reference_number` attribute is a unique identifier assigned to qualified teachers. The system validates this number to ensure it corresponds to a registered teacher.
+
+The `teacher_reference_number_validated` boolean attribute indicates whether the TRN has been successfully validated.
+
+### Participant ID Changes
+
+The `participant_id_changes` array tracks any changes to a participant's ID. Each entry includes:
+
+- `from_participant_id`: The previous participant ID
+- `to_participant_id`: The new participant ID
+- `changed_at`: Timestamp when the change occurred
+
+This tracking ensures continuity when participant records are merged or updated.
+
+### Timestamps
+
+Two important timestamps are tracked for each participant:
+
+- **updated_at**: This timestamp is updated whenever any attribute of the participant is modified. Examples of actions that trigger an update:
+  - Changing training status (defer, withdraw, resume)
```

Comment:
could we maybe talk about the models we look at as well when getting the updated at timestamp as we know its a bit complex